Grinnell-Events-iOS/Grinnell-Events-iOS/ViewController.m

```diff
@@ -187,7 +185,6 @@ -(void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
         }
         
         eventDetailViewController.theEvent = event;
-        //eventDetailViewController.title = event.title;
```

Comment:
Sarah said keep first title, remove second. I'll see if I can figure out where that's actually being set, shouldn't be too difficult.